app/src/main/java/com/isc/hermes/utils/MapConfigure.java

```diff
@@ -10,12 +12,17 @@
  */
 public class MapConfigure {
 
+    PlacesSearchedManager manager;
+    Context context;
```

Comment:
Use encapsulation in your variable please `Context context;`

Reply:
Done.